backend/generator.js

```diff
@@ -10,10 +10,10 @@ exports.executeScript = function (socket, formData) {
     return false;
   }
   var email = formData.email;
-  var gitUrl = formData.gitUrl;
+  var gitUrl = "https://github.com/" + formData.username + "/" + formData.reponame + ".git";
```

Comment:
The gitURL parameter is only used by the WEBUI part of the script. So it shouldn't affect the travis part. Yeah you would have to build the url there